indico/modules/events/payment/forms.py

```diff
@@ -36,7 +36,8 @@
 
 class AdminSettingsForm(IndicoForm):
     currencies = MultipleItemsField(_('Currencies'), [DataRequired()],
-                                    fields=(('code', _('Code')), ('name', _('Name'))),
+                                    fields=[{'id': 'code', 'caption': _('Code'), 'type': 'text'},
```

Comment:
I would really make `'type': 'text'` the default to avoid specifying it all the time